docs/insecure.md

```diff
@@ -78,9 +78,12 @@ Usually, on Ubuntu this is done with:
     cp auth/domain.crt /usr/local/share/ca-certificates/myregistrydomain.com.crt
     update-ca-certificates
 
-... and on RedHat with:
+... and on Red Hat (and its derivatives) with:
 
     cp auth/domain.crt /etc/pki/ca-trust/source/anchors/myregistrydomain.com.crt
     update-ca-trust
 
+... On some distributions, e.g. Oracle Linux 6, the Shared System Certificates feature needs to be manually enabled:
+    $ sudo update-ca-trust enable
```

Comment:
the formatting looks a bit off here.
![screen shot 2015-11-24 at 3 39 39 pm](https://cloud.githubusercontent.com/assets/922018/11384083/99d719b2-92c1-11e5-8460-0ee1269a7010.png)